Example/Example/RemoteAnimationDemoView.swift

```diff
@@ -6,6 +6,7 @@ import SwiftUI
 
 // MARK: - AnimationListView
 
+@MainActor
 struct RemoteAnimationsDemoView: View {
```

Comment:
Should't Views be main actors by default?

Reply:
I thought so too, but you get a concurrency error without this. Maybe only `body` is implicitly main-actor but not the other properties?